EventsExpress.Core/IServices/ICategoryGroupService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using EventsExpress.Core.DTOs;
+
+namespace EventsExpress.Core.IServices
+{
+    public interface ICategoryGroupService
+    {
+        IEnumerable<CategoryGroupDto> GetAllGroups();
+
+        CategoryGroupDto GetById(Guid id);
+
+        bool Exists(Guid id);
```

Comment:
Why do you need these exists methods?

Reply:
It's rather considered for the future when we can get the group by id, but Exists method is really useless, so yea, I'm on my way